lib/lotus/generators/application/container.rb

```diff
@@ -11,13 +11,16 @@ def initialize(command)
           @slice_generator     = Slice.new(command)
           @lotus_head          = options.fetch(:lotus_head)
           @lotus_model_version = '~> 0.2'
+          @app_name            = app_name
 
           cli.class.source_root(source)
         end
 
         def start
+          @app_name = ::File.basename(Dir.getwd) if @app_name == '.'
```

Comment:
This won't work if the current directory is /, however I'm not sure if this needs to be supported. Or perhaps we can have a default app name such as 'app' if the current directory is root.

What's your thoughts?